i18n/locales/en.ts

```diff
@@ -3064,7 +3064,7 @@ Stack Hint: To spend before the secret is revealed, Vanderpoole uses his signatu
       payment: 'This is your off-chain payment to Laszlo.',
       multisig:
         'This transaction will fund a multisig output between you and Laszlo.',
-      refund:
+      refund_0:
```

Comment:
Does anything else need to be changed to account for this updated key name?

Reply:
No, it was an error as a bare translation key with no string before